src/cmd/build.rs

```diff
@@ -627,7 +627,8 @@ fn do_optimization(
         // the memory in our module is imported, `wasm-opt` needs to be told that
         // the memory is initialized to zeroes, otherwise it won't run the
         // memory-packing pre-pass.
-        .arg("--zero-filled-memory");
+        .arg("--zero-filled-memory")
+        .arg("--disable-sign-ext");
```

Comment:
Would `--mvp-features` have the same effect, but also disabling mutable globals? See https://github.com/paritytech/cargo-contract/pull/891

Reply:
Good, done.